test/e2e-go/e2e_go_suite_test.go

```diff
@@ -49,8 +46,16 @@ var _ = BeforeSuite(func() {
 	tc, err = testutils.NewTestContext(testutils.BinaryName, "GO111MODULE=on")
 	Expect(err).NotTo(HaveOccurred())
 
-	By("creating a new directory")
-	Expect(tc.Prepare()).To(Succeed())
+	tc.Domain = "example.com"
+	tc.Group = "cache"
+	tc.Version = "v1alpha1"
+	tc.Kind = "Memcached"
+	tc.Resources = "memcacheds"
+	tc.ProjectName = "memcached-operator"
+	tc.Kubectl.Namespace = fmt.Sprintf("%s-system", tc.ProjectName)
+
+	By("copying sample to a temporary e2e directory")
+	Expect(exec.Command("cp", "-r", "../../testdata/go/v2/memcached-operator", tc.Dir).Run()).To(Succeed())
```

Comment:
Just an FYI we'll have to refactor this later so both v2 and v3 samples can be tested.